examples/job_results_iterator.ipynb

```diff
@@ -0,0 +1,140 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Harmony Py Library\n",
+    "### Job Results Iterator Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append('..')\n",
+    "\n",
+    "# Install harmony-py requirements.  Not necessary if you ran `pip install harmony-py` in your kernel  \n",
+    "!{sys.executable} -m pip install -q -r ../requirements/core.txt\n",
+    "\n",
+    "import concurrent.futures\n",
+    "from harmony import BBox, Client, Collection, Request, Environment\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "harmony_client = Client(env=Environment.UAT)  # assumes .netrc usage\n",
+    "\n",
+    "collection = Collection(id='C1234088182-EEDTEST')\n",
+    "request = Request(\n",
+    "    collection=collection,\n",
+    "    spatial=BBox(-165, 52, -140, 77)\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# submit an async request for processing and return the job_id\n",
+    "job_id = harmony_client.submit(request)\n",
+    "job_id\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can iterate over the results which will cause them to download in parallel while the job is running\n",
+    "# the first argument to `iterator()` is the ID of the job we want\n",
+    "# the second argument is the path where we want to download the files\n",
+    "# the third argument is whether or not we want to overwrite files - this defaults to False which\n",
+    "# speeds up processing by not re-downloading files that have already been downloaded\n",
+    "iter = harmony_client.iterator(job_id, '', True)\n",
+    "# grab all the futures that are downloading the files\n",
+    "futures = list(map(lambda x: x['path'], iter))\n",
+    "# wait for all the futures to complete\n",
+    "(done_futures, _) = concurrent.futures.wait(futures)\n",
+    "#iterate of the futures to get the file paths\n",
```

Comment:
I'm not sure what this commenting is intending to say - probably can just be deleted.

Reply:
done